NRLMMD-GEOIPS/data_fusion
# v1.9.0 data_fusion - VERSION RELEASE integration - versioning update, test output update

# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#105

# Testing Instructions
* MUST check out "main" branch on geoips and data_fusion repos.  
* THEN "pip install -e ." on both geoips and data fusion.
* You can temporarily delete all the commands in test_full_install.sh except the last "data_fusion" call - that will apply the one available data fusion test.
  * ``$GEOIPS/tests/test_full_install.sh``

# Summary
Merge 1.9.0 from main into integration branch

# Output
Successfull data_fusion call in geoips "test_full_install.sh"

Package: geoips_full
Total run time: 145 seconds
Number data types run: 1
Number data types failed: 0
